### PR TITLE
feat(argo-cd): add multi-namespace support for ApplicationSet controller RBAC

### PR DESCRIPTION
**Pull request by leehosu**
related #2919 

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

File: charts/argo-cd/templates/argocd-applicationset/clusterrolebinding.yaml [hunk 1]
```diff
@@ -13,4 +13,12 @@ subjects:
   - kind: ServiceAccount
     name: {{ template "argo-cd.applicationSet.serviceAccountName" . }}
     namespace: {{ include "argo-cd.namespace" . }}
+{{- $namespaces := index .Values.configs.params "applicationsetcontroller.namespaces" -}}
+{{- range $namespace := (split "," $namespaces) }}
+  {{- if $namespaces }}
```
Comment (yu-croco) [inline charts/argo-cd/templates/argocd-applicationset/clusterrolebinding.yaml:18]: This would be clearer ?
```suggestion
  {{- if $namespace }}
```

Comment (leehosu) [reply charts/argo-cd/templates/argocd-applicationset/clusterrolebinding.yaml]: Great! I made the changes as per your suggestion!

**Review by yu-croco — APPROVED** [review]
Thank you for your contribution. LGTM

**Review by mbevc1 — APPROVED** [review]

**Comment by spirkaa** [comment]
This change was unnecessary, ApplicationSet in any namespace feature already working beginning from chart version 6.6.0.

Additional subjects in ClusterRoleBinding do nothing, because these namespaces don't contain this service account. ApplicationSet controller uses single account in argocd namespace.

**Comment by mbevc1** [comment]
Base on the follow-up comments I'll open a revert fro this.

**Pull request merged**